app/scripts/controllers/preferences-controller.ts

```diff
@@ -463,6 +471,7 @@ export class PreferencesController extends BaseController<
   /**
    * Setter for the `useBlockie` property
    *
+   * @deprecated Use setAvatarType instead
```

Comment:
If this is used, `avatarType` and `useBlockie` will be out of sync. Maybe this should update `avatarType` as well?